### PR TITLE
UCP/API: Fix misprint in the doxygen

### PR DESCRIPTION
**Pull request by dmitrygladkov**
## What

PR fixes misprint in the doxygen for UCP/API: `ucp_tag_send_nbr()`
and + PR template is update to use italic font for comment in the HOW? section as for the WHAT? and WHY? sections. -- it seems that `_` was missed at the end of the comment.

## Why ?

`ucp_tag_send_nb()` should be used instead of `ucp_tag_send_nbr()`, because `ucp_tag_send_nbr()` is compared to `ucp_tag_send_nb()`

## How ?

`ucp_tag_send_nbr()` -> `ucp_tag_send_nb()`

### CODE REVIEW

**Comment by dmitrygladkov** [comment]
ok to test

**Comment by mellanox-github** [comment]
Can one of the admins verify this patch?

File: src/ucp/api/ucp.h [hunk 1]
```diff
@@ -2136,7 +2136,7 @@ ucs_status_ptr_t ucp_tag_send_nb(ucp_ep_h ep, const void *buffer, size_t count,
  *
  * This routine provides a convenient and efficient way to implement a
  * blocking send pattern. It also completes requests faster than
- * @ref ucp_tag_send_nbr() because:
+ * @ref ucp_tag_send_nb() because:
```
Comment (hoopoepg) [inline src/ucp/api/ucp.h]: 👍

File: PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -7,4 +7,4 @@ bug fixes why and what can be merged in a single item._
 
 ## How ?
 _It is optional but for complex PRs please provide information about the design,
-architecture, approach, etc.
+architecture, approach, etc._
```
Comment (hoopoepg) [inline PULL_REQUEST_TEMPLATE.md]: as I can see underscore here is used for Italic font (see begin of sentence)

Comment (dmitrygladkov) [reply PULL_REQUEST_TEMPLATE.md]: Yes, it seems that it (`_`) was missed.

**Comment by yosefe** [comment]
@tonycurtis @shamisp can you pls take a look?

File: PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (shamisp) [reply PULL_REQUEST_TEMPLATE.md]: Ouch :) I missed it

**Comment by shamisp** [comment]
👍 @tonycurtis ?

**Pull request merged**